proto/ibc/applications/interchain_accounts/v1/query.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+package ibc.applications.interchain_accounts.v1;
+
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "ibc/applications/interchain_accounts/v1/account.proto";
+
+option go_package = "github.com/cosmos/ibc-go/modules/apps/27-interchain-accounts/types";
+
+// Query defines the gRPC querier service.
+service Query {
+  // Query to get the address of an interchain account
+  rpc InterchainAccount(QueryInterchainAccountRequest) returns (QueryInterchainAccountResponse) {}
+}
+
+// Query request for an interchain account address
+message QueryInterchainAccountRequest {
+  option (gogoproto.equal)           = false;
+  option (gogoproto.goproto_getters) = false;
+
+  // Address is the address to query.
+  string address       = 1;
```

Comment:
What address is being specified in the request? What address is being returned in the response?

Reply:
Good point. The comment is useless here.

Reply:
I updated the comments + param names. Is it more clear?